Project.toml

```diff
@@ -0,0 +1,9 @@
+name = "GAP"
+uuid = "c863536a-3901-11e9-33e7-d5cd0df7b904"
+authors = ["Thomas Breuer <sam@math.rwth-aachen.de>", "Sebastian Gutsche <gutsche@mathematik.uni-kl.de>", "Max Horn <max@quendi.de>"]
+version = "0.1.0"
+
+[deps]
+Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
```

Comment:
`Pkg` is for building (as we need `Pkg.depots1`), and `Libdl` is actually a dependency. I found it quite interesting, too, as they both should be standard packages. However, without the explicit dependency, the packages are not found when building them into a new, clean Julia depot. That is why I added them. Actually, this might be an inconsistency in other packages. I think we are doing the right thing here.